build/commands/lib/util.js

```diff
@@ -704,8 +673,7 @@ const util = {
     restoreBraveCoreGitUrlIfGitCacheEnabled()
```

Comment:
we can drop this as well with this PR.

Reply:
@petemill please remove this before merging. I've added this workaround for the exact case we're removing here - when brave-core was synced via `gclient`.

Reply:
@goodov got a link to the workaround?

Reply:
Removed in https://github.com/brave/brave-core/pull/11543/commits/1aef34849c0e6fa8df47f013d1081a5b6a38a15f